slynk/backend/clasp.lisp

```diff
@@ -31,13 +31,18 @@
   (when (probe-file "sys:profile.fas")
     (require :profile)
     (pushnew :profile *features*))
-  (when (probe-file "sys:serve-event")
+  (when (probe-file "sys:src;lisp;modules;serve-event;")
```

Comment:
Thanks very much for all your work on this file.  I won't review it, but I trust it's the right thing every time :-)

Reply:
Ok, thanks.  Yeah,  I'll deal with issues in clasp.lisp